runtime.md

```diff
@@ -23,6 +23,9 @@ This allows the hooks to perform cleanup and teardown logic after the runtime de
 * **`bundlePath`**: (string) is the absolute path to the container's bundle directory.
 This is provided so that consumers can find the container's configuration and root filesystem on the host.
 
+* **`annotations`**: (interface) holds runtime-specified information that is not structured by this specification.
```

Comment:
On Mon, Nov 16, 2015 at 07:59:57PM -0800, Vish Kannan wrote:
> > +* **`annotations`**: (interface) holds runtime-specified
> >    information that is not structured by this specification.
> 
> What does an `interface` mean outside of golang?

Data with an unspecified schema.  E.g. void* in C [1].

[1]: https://github.com/opencontainers/specs/pull/188#issuecomment-152603357